packages/protect/README.md

```diff
@@ -106,32 +134,117 @@ if (step.getCallbacksOfType('PingOneProtectEvaluationCallback')) {
   const callback = step.getCallbackOfType('PingOneProtectEvaluationCallback');
   try {
     // Asynchronous call
-    data = await protectAPI.getData();
+    data = await protectApi.getData();
+    if (typeof data !== 'string' && 'error' in data) {
+      callback.setClientError(data.error);
+    }
   } catch (err) {
     callback.setClientError(err.message);
   }
 }
 ```
 
-## Full API
+## Quickstart with a PingOne DaVinci Flow
+
+The PingOne Protect module is intended to be used along with the DaVinci client to provide the PingOne Protect feature.
+
+### Requirements
+
+1. A PingOne environment with PingOne Protect added
+2. A worker application configured in your PingOne environment
+3. A DaVinci flow with the appropriate Protect connectors
+4. A client application with the `@forgerock/davinci-client` and `@pingidentity/protect` modules installed
+
+### Integrate into a Client Application
+
+Install both modules and their latest versions:
+
+```sh
+npm install @forgerock/davinci-client @pingidentity/protect
+```
+
+The `@pingidentity/protect` module has a `protect()` function that accepts configuration options and returns a set of methods for interacting with Protect. The two main responsibilities of the PingOne Protect module are the initialization of the profiling and data collection and the completion and preparation of the collected data for the server. You can find these two methods on the API returned by `protect()`.
+
+- `start()`
+- `getData()`
+
+When calling `protect()`, you have many different options to configure what and how the data is collected. The most important and required of these settings is the `envId`. All other settings are optional.
+
+The `start` method can be called at application startup, or when you receive the `ProtectCollector` from the server. We recommend you call `start` as soon as you can to collect as much data as possible for higher accuracy.
```

Comment:
I'd like to emphasize this a bit more than this paragraph. Can we use headers for the different approaches and place "(recommended)" in the header text for startup time approach?